niworkflows/viz/utils.py

```diff
@@ -259,7 +259,7 @@ def compose_view(bg_svgs, fg_svgs, ref=0, out_file='report.svg'):
     # Read all svg files and get roots
     svgs = [svgt.fromfile(f) for f in bg_svgs + fg_svgs]
     roots = [f.getroot() for f in svgs]
-    nsvgs = len(svgs) // 2
+    nsvgs = len(bg_svgs)
```

Comment:
This was the main issue. With only `bg_svgs`, `nsvgs == 1`, so on the first round through line 286, `yoffset` was set to 0.

Because `len(fg_svgs) in (0, len(bg_svgs)`, this shouldn't change the two-layer functioning.